CHANGELOG.md

```diff
@@ -2843,7 +2845,7 @@ FEATURES:
 * Better key output, pubkey go-amino hex bytes now output by default
 * gaiad init overhaul
   * Create genesis transactions with `gaiad init gen-tx`
-  * New genesis account keys are automatically added to the client keybase (introduce `--client-home` flag)
+  * New genesis account keys are automatically added to the client keybase (introduce `--home-client` flag)
```

Comment:
```suggestion
  * New genesis account keys are automatically added to the client keybase (introduce `--keyring-dir` flag)
```

Reply:
That's actually ancient news and should not be changed.  But I accidentally changed it.  It should remain `--client-home`.